### PR TITLE
added navigation to all pages

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
Added Nav and footer to all pages
Made some small fixes in forms
Fixed CSS links in all pages

### CODE REVIEW

**Comment by BelideNikhil** [comment]
image

Rest looks awesome!

**Review by BelideNikhil — APPROVED** [review]

**Comment by poornima-alugubelly** [comment]
> image
> 
> Rest looks awesome!

Hey thank u! fixed it

File: index.html [hunk 1]
```diff
@@ -31,26 +26,25 @@
 					</li>
 				</ul>
 			</div>
+			<div class="search-bar input">
+				<input type="text" />
+				<button>
```
Comment (MiheerTamkhane) [inline index.html:31]: try input type : "search".

**Review by MiheerTamkhane — APPROVED** [review]
image

The range input has some problem i guess?

File: index.html (hunk 1 above)
Comment (poornima-alugubelly) [reply index.html:31]: I didn't know that such a type is there thank u!

**Comment by poornima-alugubelly** [comment]
> image
> 
> The range input has some problem i guess?

not a problem, JS is needed to make the filled in part move according to the value chosen, for now I have given a fixed value so it is half filled

**Pull request merged**